addons/test_mail_full/tests/test_portal.py

```diff
@@ -295,45 +279,53 @@ def test_customer_access_logged_without_access(self):
         self.authenticate(self.env.user.login, self.env.user.login)
         res = self.url_open(self.record_access_url)
         self.assertEqual(res.status_code, 200)
-        self.assert_URL(res.url, f'/my/test_portal/{self.record_portal.id}')
+        parsed_url = url_parse(res.url)
+        self.assertEqual(parsed_url.path, f'/my/test_portal/{self.record_portal.id}')
```

Comment:
It's not a web url, it shouldn't be modify

Reply:
I've made this changes as i am not using `assert_URL` anymore.